projects/capstone/Dockerfile

```diff
@@ -16,7 +16,8 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder-python
 RUN apt-get update && apt-get install -y make cmake
-RUN git clone --depth 1 --branch v5 https://github.com/aquynh/capstone.git capstonev5
-RUN git clone --depth 1 --branch next https://github.com/aquynh/capstone.git capstonenext
+RUN pip3 install --upgrade setuptools build wheel
```

Comment:
You might want to consider pinning these to a known stable version. It'll mean manual version bumps, but might prevent or allow easier debugging of unexpected build failures in the future.

Reply:
Good point, we will take care of it.